test/unit/mocked_data/test_get_bgp_neighbors/normal/expected_result.json

```diff
@@ -1 +1,180 @@
+{
+  "global": {
+    "router_id": "10.255.255.1",
+    "peers": {
+      "192.168.0.0": {
+        "description": "",
+        "remote_id": "10.255.255.240",
+        "address_family": {
+          "ipv6": {
+            "received_prefixes": -1,
+            "sent_prefixes": -1,
+            "accepted_prefixes": -1
+          },
+          "ipv4": {
+            "received_prefixes": 2,
+            "sent_prefixes": 9,
+            "accepted_prefixes": 2
+          }
+        },
+        "remote_as": 30000,
+        "is_up": true,
+        "local_as": 30001,
+        "is_enabled": true,
+        "uptime": 699262
+      },
+      "10.3.1.254": {
+        "description": "",
+        "remote_id": "10.48.71.105",
+        "address_family": {
+          "ipv6": {
+            "received_prefixes": -1,
+            "sent_prefixes": -1,
+            "accepted_prefixes": -1
+          },
+          "ipv4": {
+            "received_prefixes": 7,
+            "sent_prefixes": 4,
+            "accepted_prefixes": 7
+          }
+        },
+        "remote_as": 29999,
+        "is_up": true,
+        "local_as": 30001,
+        "is_enabled": true,
+        "uptime": 12688391
+      },
+      "192.169.1.0": {
+        "description": "",
+        "remote_id": "",
+        "address_family": {
+          "ipv6": {
+            "received_prefixes": -1,
+            "sent_prefixes": -1,
+            "accepted_prefixes": -1
+          },
+          "ipv4": {
+            "received_prefixes": -1,
+            "sent_prefixes": -1,
+            "accepted_prefixes": -1
+          }
+        },
+        "remote_as": 30000,
+        "is_up": false,
+        "local_as": 30001,
+        "is_enabled": true,
+        "uptime": 10808600
```

Comment:
Discussed in private already.
For the record: this is the way Juniper provides, when a neighbor was up and then goes down, it does not reset the timer. People tend to prefer one or the other (the timer indeed provides you more info, but creates a bit of confusion given that Cisco resets this timer).
For the moment, we pushed a documentation note, but ideally, we should have two separate keys, e.g. `uptime` and `last_uptime`.